File: 이건희/level2/숫자의 표현.py
def solution(n):
    result = 1
    cnt = 0
    num = n // 2 + 1
    
    for i in range(1, num):
        for j in range(i, num + 1):
            cnt += j
            if cnt == n:
                result += 1
                cnt = 0
            elif cnt > n:
                cnt = 0
                break
            
    return result

File: 이훈일/Programmers/Level2/숫자의 표현.java
class Solution {
    public int solution(int n) {
        int result = 1;
        
        for(int i = 1; i <= n/2; i++) {
            int sum = 0;
            int num = i;
            
            while(sum < n) {
                sum += num++;
            }
    
            if(sum == n) {
                result++;
            }
        }
        
        return result;
    }
}
